R/runCalibration.R

```diff
@@ -216,6 +216,9 @@ runCalibrationLogit <- function(path,
 
     totalStep <- select(stepSizeParams, "region", "loc", "typ", "inc")
 
+    # Set starting point for all GAMS runs in this iteration to output of previous iteration
+    copyInitialGdx(path, file.path(path, paste0("calibration_", i - 1, ".gdx")), overwrite = TRUE)
```

Comment:
Is there an advantage over using `output.gdx`?
```suggestion
    copyInitialGdx(path, gdxOutput, overwrite = TRUE)
```